src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/ExportProjectNodeComServiceAttribute.cs

```diff
@@ -24,11 +24,6 @@ public string[] Iid
             get;
         }
 
-        public static bool IncludeInherited
```

Comment:
This is required, it's part of the metadata of the contact. Without it, MEF won't find these types.

Reply:
Reinstated.